src/app/qgsattributetypedialog.h

```diff
@@ -63,6 +64,26 @@ class APP_EXPORT QgsAttributeTypeDialog: public QDialog, private Ui::QgsAttribut
     bool labelOnTop() const;
 
     /**
+     * Setter for lable alias
+     */
+    void setAlias( const QString &alias );
+
+    /**
+     * Getter for lable alias
+     */
+    QString alias();
```

Comment:
Const,

Sidenote: method documentation should normally provide more information than implicitly available from the method name (i.e. do not state the obvious). It's not critical in here since it's in app, so no public api.